crates/wasi-preview1-component-adapter/Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [package]
 name = "wasi-preview1-component-adapter"
-version.workspace = true
+version = "23.0.0+wasi-0.2.0"
```

Comment:
While having an explicit version here improves things my worry is that this will basically never get updated, even when it needs to. Currently all other version management in this repository is automated so I would expect that this would never change as a result.

I can think of two other primary alternatives for continuing to embed version information in the adapter:

* Update the bump-version script/CI configuration to additionally rebuild the adapters. I find this not super appealing though because it adds binary file churn to this repository which feels unfortunate.
* Add a `build.rs` to the provider crate which injects version information. To be maximally useful this would probably want to have zero build-dependencies which makes it a bit difficult to do, though.

Overall I'd probably still be most in favor of just dropping the version information in the binary. AFAIK we haven't really ended up using it all that much in practice.

Reply:
Ok, let's just remove the version info (for now) then